### DISCUSSION TITLE
NASA 撞歪小行星！

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1039] [General]
不知道大家有沒有看過電影<地心引力>
雖然是一部蠻久的電影了
裡面的劇情讓人見識到太空碎片的撞擊威力也體認了太空環境的危險
但真的好好看，跟一般的太空災難片完全不同，非常推推！

我也想起之前看過的這篇報導
難以想像要是小行星撞擊真的發生在地球上會造成什麼後果
NASA去年以人造衛星將一顆小行星撞歪
想試驗是否能應用在近地小行星撞擊的預防上
這是人類第一次改變成成功改變星體軌道，好有趣

雖然這次試驗僅選擇了一顆無害的小行星
但不知道這樣下去任由人類排列組合的星體軌道會不會產生其他負面影響
https://tw.news.yahoo.com/news/nasa%E6%92%9E%E6%AD%AA%E5%B0%8F%E8%A1%8C%E6%98%9F%E8%A9%A6%E9%A9%97%E6%88%90%E5%8A%9F-%E8%BB%8C%E9%81%93%E9%80%B1%E6%9C%9F%E7%B8%AE%E7%9F%AD%E6%AF%94%E9%A0%90%E6%9C%9F%E5%A4%9A22%E5%88%86%E9%90%98-011400920.html

google 也為這件事藏了彩蛋哈哈，每次都會被google的彩蛋可愛到！
https://news.cts.com.tw/cts/general/202209/202209292093549.html

**Comment by dabbie109303501** [comment]
若是之後真的能研究出讓小行星真正改變軌道的方法，就不用擔心有一天會像以前毀滅恐龍造成冰河時期的慘案發生了

**Comment by ru0372572** [comment]
我覺得文章提到 : 要能夠識別更多種類的太空岩石也很重要，因為確定結構與質料才能確保每一次的撞擊都能達到我們想要的結果。

**Comment by wenchi9023** [comment]
第一次聽到這項技術，好酷!
但如果每顆小行星都有可能影響彼此，人們將自以為無害的小行星軌道偏移後是否會帶來負面的連帶效應呢?
感覺需要更多的研究和評估

**Comment by harrylee1971** [comment]
或許未來就會有遇到這種技術的時候，因為世界那麼大，甚麼事情都不奇怪阿哈哈哈哈哈

**Comment by Jzd314** [comment]
偶爾會看到一些小行星近距離經過地球的新聞，說不定這種技術在未來真的會拯救地球

**Comment by 110302550** [comment]
未來一直在進步，地球已經沒什麼可發展了，來就應該朝外發展，但還是好期待未來會發展成甚麼樣子。

**Comment by wanderer657** [comment]
在太空站上，宇航員們每天都在忙碌地進行著各種科學實驗和任務，他們的生活雖然充滿挑戰，但也充滿了希望和探索的樂趣。其中，有一位名叫艾莉斯的宇航員，是這次任務的領頭人，她有著堅定的意志和卓越的領導能力。

然而，一場突如其來的危機打破了他們平靜的生活。小行星的軌道突然受到了人類的操控，這是NASA實驗的一部分，但不幸的是，實驗出現了意外。

艾莉斯立即召集了其他宇航員，制定了應對計劃。他們迅速穿上太空服，準備進行緊急撤離。此時，太空站已經開始顫抖，星體的軌道也變得混亂不堪。

眼看著小行星朝著太空站高速撞來，艾莉斯的心情變得焦急，但她絲毫不敢放棄，她知道宇航員們的生命正處於危險之中。她發出了指令，帶領大家一同使用火箭背包進行遠離太空站的逃生。

在慌亂的情況下，艾莉斯始終保持著冷靜和堅定，她不斷給予其他宇航員信心和鼓勵。每一個人都全力以赴，盡全力逃離了太空站，然而小行星的撞擊速度依然驚人，太空站在一陣轟鳴聲中被摧毀成碎片。

在太空中漂流時，艾莉斯始終保持著頭腦清晰，她利用自己的經驗和技能指揮著大家，讓他們盡可能地遠離撞擊範圍。即使面臨著生死存亡的考驗，她也沒有放棄，堅信著大家一定能夠度過這次危機。

最終，在艾莉斯的領導下，所有宇航員成功地脫離了小行星的撞擊範圍，他們的生命得到了拯救。雖然太空站已經成為了一片廢墟，但每個人都感到由衷的欣慰和感激。這次危機讓他們更加意識到太空探索的危險性，也讓他們更加珍惜生命的可貴。
